packages/core-app-api/src/app/types.ts

```diff
@@ -321,7 +321,7 @@ export type AppContext = {
   /**
    * Get a list of all plugins that are installed in the app.
    */
-  getPlugins(): BackstagePlugin<any, any>[];
+  getPlugins(): BackstagePlugin<any, any, any>[];
```

Comment:
Did you investigate any way to avoid this change btw? perhaps worth moving it to just `BackstagePlugin`?

Reply:
But `PluginInputOptions` is used between 2 types: `BackstagePlugin` and `PluginConfig`
![Screenshot 2022-07-07 at 12 30 23](https://user-images.githubusercontent.com/2265094/177753487-00162ef6-a7f5-408c-b753-32b386df9bfa.png)